scripts/check_compliance.py

```diff
@@ -216,26 +231,28 @@ def run(self):
         os.environ["KCONFIG_STRICT"] = "y"
 
         try:
-            kconf = kconfiglib.Kconfig()
+            return kconfiglib.Kconfig()
         except kconfiglib.KconfigError as e:
             self.add_failure(str(e))
-            return
-
-        #
-        # Look for undefined symbols
-        #
+            raise EndTest
```

Comment:
```python
self.error(str(e))
```
?

Reply:
Problem is `error()` messages aren't shown, and this one will point to the file/line of the problem, so it's pretty important. Dunno about shippable though.

Could view it as a Kconfig pre-test too. Do the files parse? :P

Reply:
The message might also be pretty long if it's from Kconfiglib, so might be nice to have it in the body.

Reply:
> Problem is error() messages aren't shown

Do you mean on github? I see them on the command line.

Reply:
Yup, on GitHub (iirc it just says "error blah blah please report this"), and maybe Shippable.

Errors in the Kconfig files aren't a problem with the test itself, so I think the `add_failure()` kinda makes sense there.